src/i18n/messages-types.ts

```diff
@@ -156,6 +156,9 @@ export interface I18nFormatArgTypes {
       "unit": string | number;
     }
   }
+  "drawer": {
+    "loadingText": never;
```

Comment:
What does `never` mean here? Is it temporary while the strings are not avaialble?

Reply:
If you look at this file, most of the types are `never`. Although I missed that this file is auto-generated, so I removed this update in this PR.